libs/blog-bff/newsletter/src/lib/api.ts

```diff
@@ -0,0 +1,43 @@
+import { Hono } from 'hono';
+import { csrf } from 'hono/csrf';
+import { z } from 'zod';
+
+import { wpNewsletterClientMw } from '@angular-love/util-wp';
+
+const app = new Hono();
+
+app.use(
+  '*',
+  csrf({
```

Comment:
Can we enable `csrf` globally? Or should it be used inly for newsletter sign up?

Reply:
As far as I know we only need this for newsletter sign up